### PR TITLE
Create examining-power.md

### PR DESCRIPTION
**Pull request by gigikenneth**
### Summary
Added Chapter 1 on Examining Power to Data Feminism folder

Fixes #<NUM>

No fixes here

### List of changes proposed in this PR (pull-request)

* New sub-chapter to the Data Feminism section.

### What should a reviewer concentrate their feedback on?

- [ ] Does the content look okay?
- [ ] Are the links okay?
- [ ] Does the structure of the summary make sense? 

### Acknowledging contributors

- [x] All contributors to this pull request are already named in the [table of contributors](https://github.com/the-turing-way/the-turing-way/blob/main/README.md#contributors) in the README file.
- [ ] The following people should be added to the [table of contributors](https://github.com/the-turing-way/the-turing-way/blob/main/README.md#contributors) in the README file:

### CODE REVIEW

File: book/website/ethical-research/data-feminism/examining-power.md [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+## Key Points
+- **Understanding Influence**: It is essential to recognize how racism, sexism, and privilege shape oppression.
+- **Naming Oppression**: Examining power involves identifying and explaining the oppressive forces ingrained in our daily lives.
+
+(er-data-feminism-c1-powerdomains)=
+## Domains of Power
+Power systems can be configured and experienced across four domains:
+
+![image](https://hackmd.io/_uploads/BJeHBss40.png)
+*Source: https://data-feminism.mitpress.mit.edu/pub/vi8obxh7#nkiyehtj87r*
```
Comment (JimMadge) [inline book/website/ethical-research/data-feminism/examining-power.md:17]: Could you follow the [style guide for figures](https://book.the-turing-way.org/community-handbook/style/style-figures#using-figures-in-the-turing-way) in the Community Handbook?

As it is a table, but might be better to reproduce it in the book [as a table](https://jupyterbook.org/en/stable/reference/cheatsheet.html#tables) and include a reference.

File: book/website/ethical-research/data-feminism/examining-power.md [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+(er-data-feminism-c1-intersectionality)=
+## Intersectionality
+Dimensions such as gender, race, sexuality, geography, and ability can lead to unjust oppression or unearned privilege across these four domains.
+
+(er-data-feminism-c1-questions)=
+## Critical Questions
+To understand how power unfolds in and around data, consider:
+1. **Who is engaged in data science work (and who is not)?**
+2. **Whose goals are prioritized in data science (and whose are not)?**
+3. **Who benefits from data science (and who is overlooked or harmed)?**
```
Comment (JimMadge) [inline book/website/ethical-research/data-feminism/examining-power.md:33]:
```suggestion
To understand how power unfolds in and around data, consider:

1. **Who is engaged in data science work (and who is not)?**
2. **Whose goals are prioritised in data science (and whose are not)?**
3. **Who benefits from data science (and who is overlooked or harmed)?**
```

Lists can fail to render if there isn't a blank line before and after 🙂 .

File: book/website/ethical-research/data-feminism/examining-power.md [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+
+
+Researchers have shown that [images of immigrants, abused children, and deceased individuals have been used to train software without consent](https://slate.com/technology/2019/03/facial-recognition-nist-verification-testing-data-sets-children-immigrants-consent.html), raising serious ethical issues.
+
+(er-data-feminism-c1-datagoals)=
+## Current Data Goals
+Data science is predominantly used for:
+1. **Profit (for a few)** - seen in corporations
+2. **Surveillance (of the marginalized)** - used by governments
+3. **Efficiency (amidst data scarcity)** - prioritized by research institutions/universities such as optimizing algorithms and models to work with limited data
```
Comment (JimMadge) [inline book/website/ethical-research/data-feminism/examining-power.md:57]:
```suggestion
Data science is predominantly used for:

1. **Profit (for a few)** - seen in corporations
2. **Surveillance (of the marginalised)** - used by governments
3. **Efficiency (amidst data scarcity)** - prioritised by research institutions/universities such as optimising algorithms and models to work with limited data
```

File: book/website/ethical-research/data-feminism/examining-power.md [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+
+(er-data-feminism-c1-goals)=
+## The Goal of Examining Power
+The essence of examining power is not only to understand power but also to challenge and change it.
+
+(er-data-feminism-c1-casestudies)=
+## Case Studies
+- **[Serena Williams and the Healthcare System](https://edition.cnn.com/2018/02/20/opinions/protect-mother-pregnancy-williams-opinion/index.html)**: Illustrates racial bias in healthcare.
+- **[Joy Buolamwini and MIT's Facial Recognition System](https://news.mit.edu/2018/study-finds-gender-skin-type-bias-artificial-intelligence-systems-0212)**: Highlights issues of bias in technology.
+- **[CloudWalk and Zimbabwe](https://qz.com/africa/1287675/china-is-exporting-facial-recognition-to-africa-ensuring-ai-dominance-through-diversity)**: Raises ethical concerns about data sharing and human rights.
```
Comment (JimMadge) [inline book/website/ethical-research/data-feminism/examining-power.md:67]:
```suggestion
## Case Studies

- **[Serena Williams and the Healthcare System](https://edition.cnn.com/2018/02/20/opinions/protect-mother-pregnancy-williams-opinion/index.html)**: Illustrates racial bias in healthcare.
- **[Joy Buolamwini and MIT's Facial Recognition System](https://news.mit.edu/2018/study-finds-gender-skin-type-bias-artificial-intelligence-systems-0212)**: Highlights issues of bias in technology.
- **[CloudWalk and Zimbabwe](https://qz.com/africa/1287675/china-is-exporting-facial-recognition-to-africa-ensuring-ai-dominance-through-diversity)**: Raises ethical concerns about data sharing and human rights.
```

File: book/website/ethical-research/data-feminism/examining-power.md [hunk 5]
```diff
@@ -0,0 +1,74 @@
...
+
+(er-data-feminism-c1-reading)=
+## Further Reading
+- **[Algorithms of Oppression by Safiya Umoja Noble](https://www.goodreads.com/book/show/34762552-algorithms-of-oppression)**: Explores how gender and racial biases in information systems are complex and often rooted in the data and models created by small, homogenous groups.
+- **[Artificial Unintelligence by Meredith Broussard](https://www.goodreads.com/book/show/36722634-artificial-unintelligence)**: Goes into the limitations and misunderstandings embedded within AI systems.
+- **[Invisible Women by Caroline Criado Pérez](https://www.goodreads.com/book/show/41104077-invisible-women)**: Discusses gender biases in data.
+
```
Comment (JimMadge) [inline book/website/ethical-research/data-feminism/examining-power.md:74]:
```suggestion
## Further Reading

- **[Algorithms of Oppression by Safiya Umoja Noble](https://www.goodreads.com/book/show/34762552-algorithms-of-oppression)**: Explores how gender and racial biases in information systems are complex and often rooted in the data and models created by small, homogenous groups.
- **[Artificial Unintelligence by Meredith Broussard](https://www.goodreads.com/book/show/36722634-artificial-unintelligence)**: Goes into the limitations and misunderstandings embedded within AI systems.
- **[Invisible Women by Caroline Criado Pérez](https://www.goodreads.com/book/show/41104077-invisible-women)**: Discusses gender biases in data.
```

File: book/website/ethical-research/data-feminism/examining-power.md [hunk 6]
```diff
@@ -0,0 +1,74 @@
+(er-data-feminism-c1)=
+
+# Chapter 1: Examining Power
+
+Here are some main ideas from this chapter on examining power. 
+
+(er-data-feminism-c1-keypoints)=
+## Key Points
+- **Understanding Influence**: It is essential to recognize how racism, sexism, and privilege shape oppression.
+- **Naming Oppression**: Examining power involves identifying and explaining the oppressive forces ingrained in our daily lives.
```
Comment (JimMadge) [inline book/website/ethical-research/data-feminism/examining-power.md:10]:
```suggestion
## Key Points

- **Understanding Influence**: It is essential to recognise how racism, sexism, and privilege shape oppression.
- **Naming Oppression**: Examining power involves identifying and explaining the oppressive forces ingrained in our daily lives.
```

**Review by JimMadge — CHANGES_REQUESTED** [review]
Looks good.

I've looked mainly at the format over the content.
The build is failing because the new page isn't in the table of contents.
You can read about how to add it [here](https://book.the-turing-way.org/community-handbook/contributing/contributing-workflow.html?highlight=table%20contents#add-your-new-files-to-the-book-s-table-of-contents)

File: book/website/ethical-research/data-feminism/examining-power.md (hunk 1 above)
Comment (gigikenneth) [reply book/website/ethical-research/data-feminism/examining-power.md]: Hi! The matrix wasn't particularly necessary as everything was listed out anyway, so I took it out.

**Comment by Arielle-Bennett** [comment]
Hi Gigi, you've opened a new PR to update the TOC https://github.com/the-turing-way/the-turing-way/pull/3715 

However, if you go back into the branch `data-feminism-chapter1` (that adds the changes you're proposing above) you should be able to add the toc change the to the same branch which will make merging easier because they both won't show as broken - happy to walk through this later! 

Essentially, they're both failing because the two PRs don't know about each other: they just see some partial changes which won't work with the structure of the book in isolation - a chapter without a TOC addition, and a TOC update which doesn't point to a chapter.

File: book/website/ethical-research/data-feminism/data-feminism-power.md [hunk 7]
```diff
@@ -0,0 +1,76 @@
...
+## Current Data Goals
+Data science is predominantly used for:
+
+1. **Profit (for a few)** - seen in corporations
+2. **Surveillance (of the marginalised)** - used by governments
+3. **Efficiency (amidst data scarcity)** - prioritised by research institutions/universities such as optimising algorithms and models to work with limited data
+
+(er-data-feminism-c1-goals)=
+## The Goal of Examining Power
+The essence of examining power is not only to understand power but also to challenge and change it.
```
Comment (SaranjeetKaur) [inline book/website/ethical-research/data-feminism/data-feminism-power.md:62]:
```suggestion

The essence of examining power is not only to understand power but also to challenge and change it.
```

**Comment by gigikenneth** [comment]
Hi @JimMadge, thank you for your help! I think the pull request is good to go now? 👀

**Review by JimMadge — APPROVED** [review]

**Review by SaranjeetKaur — APPROVED** [review]
This looks good! Happy to approve!

**Comment by JimMadge** [comment]
@all-contributors please add @SaranjeetKaur for review

**Pull request merged**